go.mod

```diff
@@ -1,8 +1,16 @@
 module github.com/oiime/logrusbun
 
-go 1.16
+go 1.20
 
 require (
-	github.com/sirupsen/logrus v1.8.1
-	github.com/uptrace/bun v0.3.9
+	github.com/sirupsen/logrus v1.9.0
+	github.com/uptrace/bun v1.1.12
```

Comment:
Unless there was a breaking change in `bun` API there is not reason to require a newer version, the package version here indicates the minimum version required